.zuul.yaml

```diff
@@ -96,6 +96,7 @@
     description: |
       Run the UI Graphing test and tempest smoketests on osp18.
     vars:
+      crc_enable_monitoring: true
```

Comment:
Wait, does crc_enable_monitoring work with extra-vars?
If yes, then this should eventually be migrated into the graphing deployment vars file in https://github.com/openstack-k8s-operators/telemetry-operator/pull/696 (CC: @MiguelCarpio )
If no, then this is fine as-is and needs no updates.

Reply:
TBH don't know :confused: